svm/src/transaction_processor.rs

```diff
@@ -2498,7 +2498,7 @@ mod tests {
     fn test_validate_transaction_fee_payer_is_nonce() {
         let lamports_per_signature = 5000;
         let rent_collector = RentCollector::default();
-        let compute_unit_limit = 2 * solana_compute_budget_program::DEFAULT_COMPUTE_UNITS;
+        let compute_unit_limit = 2 * solana_loader_v4_program::DEFAULT_COMPUTE_UNITS;
```

Comment:
Why this change? Aren't the numeric values different 150 vs 2000?

Reply:
The test doesn't seem to care about the actual value.

Reply:
The change is also to remove dep on `programs/compute-budget` crate.

Reply:
If the value is irrelevant, why use the constant at all?

Reply:
Yeah, good question!
Not sure if it needs a specific lower bound for the value. I'll try.

Reply:
The test seems to use this value for the priority fee. So any value would have worked. I've removed the use of the constant.